Source/WebCore/page/SecurityOrigin.h

```diff
@@ -225,13 +225,14 @@ class SecurityOrigin : public ThreadSafeRefCounted<SecurityOrigin> {
     SecurityOriginData m_data;
     String m_domain;
     String m_filePath;
-    bool m_universalAccess { false };
-    bool m_domainWasSetInDOM { false };
-    bool m_canLoadLocalResources { false };
-    bool m_enforcesFilePathSeparation { false };
-    bool m_needsStorageAccessFromFileURLsQuirk { false };
+    bool m_universalAccess : 1 { false };
```

Comment:
What's the size before and after this change? The commit log should mention this. Otherwise, the benefit is unclear.